hometask-2-OOP/src/Inventory.ts

```diff
@@ -1 +1,22 @@
-// your code goes here
\ No newline at end of file
+import { ItemComparator } from './interfaces';
+import { Item } from './Item';
+
+export class Inventory {
+	private items: Item[];
+
+	public addItem(item: Item): void {
+		this.items.push(item);
+	}
+
+	public sort(comparator?: ItemComparator): void {
```

Comment:
could you please implement method overload here?

Reply:
Sure, added.